packages/devtools_app/lib/src/screens/memory/panes/profile/profile_pane_controller.dart

```diff
@@ -24,7 +24,7 @@ class ProfilePaneController extends DisposableController
   /// Specifies if the allocation profile should be refreshed when a GC event
   /// is received.
   ValueListenable<bool> get refreshOnGc => _refreshOnGc;
-  final _refreshOnGc = ValueNotifier<bool>(true);
+  final _refreshOnGc = ValueNotifier<bool>(false);
```

Comment:
Leave a comment referencing https://github.com/flutter/devtools/issues/5176 and stating that this should be turned back on by default once it's resolved.